src/mumble/MainWindow.cpp

```diff
@@ -741,6 +742,36 @@ ClientUser *MainWindow::getContextMenuUser() {
 	return nullptr;
 }
 
+std::tuple< ClientUser *, Channel * > MainWindow::getContextMenuTargets() {
```

Comment:
I think a dedicated struct for the return value probably increases code readability:
```cpp
struct ContextMenuTarget {
  ClientUser *user = nullptr;
  Channel *channel = nullptr;
};
```

Reply:
For future reference, this would probably make a good entry point for #3897